packages/firestore/src/local/mutation_queue.ts

```diff
@@ -73,8 +73,6 @@ export interface MutationQueue extends GarbageSource {
 
   /**
    * Creates a new mutation batch and adds it to this mutation queue.
-   *
-   * TODO(multitab): Make this operation safe to use from secondary clients.
```

Comment:
Should this have "Multi-Tab Note: This operation is safe to use from secondary clients." ?  (please check all the comments for consistency)

Reply:
Hm, I have added this comment in a few places, but forgot a bunch. Since most operations are now multi-tab safe, should we reverse the comment and put it on the few methods that are not safe to use from secondary tabs?

Reply:
SGTM.

Reply:
Done